assets/wizards/googleAdManager/views/manageAdSlotsScreen.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Ad Manager Ad Slot Management Screens.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ActionCard,
+	CheckboxControl,
+	withWizardScreen,
+} from '../../../components/src';
+
+/**
+ * Subscriptions management screen.
+ */
+class ManageAdSlotsScreen extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const {
+			adSlots,
+			onClickDeleteAdSlot,
+		} = this.props;
+
+		return (
+			<div className="newspack-manage-ad-slots-screen">
```

Comment:
Since this Wizard doesn't have any styles, this `div` isn't really necessary, I think? You could replace with a `Fragment` and simplify the DOM a little bit.